packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,155 @@
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+There are several open questions with the current visual design. They're tracked in this GitHub issue [#1399](https://github.com/ni/nimble/issues/1399).
+
+## Design
+
+Accordion group, appearance ghost
+
+```html
+<nimble-accordion appearance="ghost">
+    <nimble-accordion-item>
+        <span slot="heading">Accordion one</span>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion one content
+    </nimble-accordion-item>
+    <nimble-accordion-item expanded>
+        <span slot="heading">Accordion two</span>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion two content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <span slot="heading">Accordion three</span>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion three content
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+Accordion group, appearance block with a nested accordion
+
+```html
+<nimble-accordion appearance="block">
+    <nimble-accordion-item>
+        <div slot="heading">Accordion one</div>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion one content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion two</div>
+        <nimble-button>Text Button</nimble-button>
+        Accordion two content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion three</div>
+        <nimble-checkbox></nimble-checkbox>
+        <nimble-accordion>
+            <nimble-accordion-item>
+                <div slot="heading">Nested Accordion One</div>
+                Nested Accordion One Content
+            </nimble-accordion-item>
+            <nimble-accordion-item>
+                <div slot="heading">Nested Accordion Two</div>
+                Nested Accordion Two Content
+            </nimble-accordion-item>
+        </nimble-accordion>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion four</div>
+        <nimble-button>Text Button</nimble-button>
+        Accordion four content
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+    -   `expand-mode` - `multi`
```

Comment:
If there is only a single expand-mode option then we can avoid having the property at all for now.

Instead there should be a note here stating that expand-mode will be omitted / unconfigurable from the FAST base class.